job_traveler/docevents/job_traveler.py

```diff
@@ -1,4 +1,17 @@
 import frappe
+from erpnext.stock.doctype.item.item import get_item_details
+
+
+def cutomer_part_number(item, current_cutomer):
```

Comment:
`cutomer` ???? Let's rename this to `_get_customer_part_name` so that its clearer that its is an internal function and does not clash with `get_customer_part_name` in the `queries` namespace.